Source/EasyNetQ/DelayedExchangeScheduler.cs

```diff
@@ -76,6 +76,8 @@ IMessageDeliveryModeStrategy messageDeliveryModeStrategy
             var properties = new MessageProperties();
             if (publishConfiguration.Priority != null)
                 properties.Priority = publishConfiguration.Priority.Value;
+            if (publishConfiguration.Headers != null)
```

Comment:
```suggestion
            if (publishConfiguration.Headers?.Count > 0)
```
here and in other places